### DISCUSSION TITLE
Is it possible to log it conditionally?

### DISCUSSION THREAD

**Discussion by ghost** [KyleRoss/node-lambda-log#74] [Q&A]
Hello,

I'm using middy, basically forwarding the request from a chain of middlewares, and I want to track the input, and the output that each middleware produces, however, I want to log it if and only if it errors out, i.e., caught by the error handler.

Is this possible with this library or do I need to implement it by myself?

**Comment by KyleRoss** [comment]
I do not see why it wouldn't work in this situation. I have not implemented anything that will allow plug and play functionality with middy, so you'd have the implement LambdaLog into your middleware manually, but it should be rather simple!
